### ISSUE TITLE
Full working sample

### ISSUE THREAD

**Issue by realn3rd** [phonegap/phonegap-plugin-contentsync#137]
Can anyone give an full working example for downloading, extracting und redirecting to the extracted content?

**Comment by imhotep** [comment]
Yes [here](https://github.com/phonegap/phonegap-plugin-contentsync/tree/master/sample)

**Comment by realn3rd** [comment]
It work's finde, but when i restart the app, the old local content appears.

**Comment by imhotep** [comment]
Right. The plugin doesn't manage content for you. It just syncs content to stores it to some permanent location. There are different ways you can manage content and it's up to you how you want to do that.

**Comment by realn3rd** [comment]
How can I add a simple test routine to the app?

**Comment by realn3rd** [comment]
any idea?

**Comment by imhotep** [comment]
@realn3rd what do you mean by simple test routine ?

**Comment by realn3rd** [comment]
A simple way to test on app start if content were already downloaded und redirect to this content.

**Comment by realn3rd** [comment]
Maybe something at device-ready-event?

**Comment by realn3rd** [comment]
Nice, but it doesn't work :/

**Comment by imhotep** [comment]
What doesn't work ? Could you be more specific ?

**Comment by realn3rd** [comment]
The App shows only the local content, not the cached after restarting it.

**Comment by imhotep** [comment]
@realn3rd did you set `document.location=data.localpath`in the `complete` callback ?

**Comment by realn3rd** [comment]
Yes
[http://pastebin.com/hsRCNKzA](http://pastebin.com/hsRCNKzA)

**Comment by realn3rd** [comment]
Any idea?

**Comment by imhotep** [comment]
@realn3rd sorry for delay. You can try to use the following (on iOS 9+ and above) if you're using WKWebView:

`ContentSync.loadUrl(YOUR_URL)`

I've updated the sample in the repository to reflect this.
